### PR TITLE
Added a blog post for GSoC

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by Holzhaus** [comment]
Converted to draft because this needs #265 to be merged first. Then this PR can be rebased.

**Comment by daschuer** [comment]
OK, this is now again ready for review.

File: content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+This year the program is open for all new or beginner contributors of Mixxx. It is no longer limited to enrolled students. New contributors have the opportunity to work on Mixxx full-time as a 12+ weeks programming project and receive a stipend sponsored by Google. For more information, check out the [Google Summer of Code Contributor Guide](https://google.github.io/gsocguides/student/).
+
+You will find the [GSoC 2022 Project Ideas for Mixxx](https://github.com/mixxxdj/mixxx/wiki/gsoc-2022-ideas) in our wiki, but we're also interested in project ideas that matters more to you. Taking a look at our [GSoC advice page](https://github.com/mixxxdj/mixxx/wiki/gsocadvice) is recommended if you're planning to apply. The best way to get started is to participate in the Mixxx community, so have a look at our [GitHub repository](https://github.com/mixxxdj/mixxx) and consider making a small code contribution to show that you're able to familiarize yourself with our codebase.
+
+The official application period will start **April 4**. Potential GSoC contributors should start now to discuss application ideas here on Zulip. Just start a new topic in the #gsoc stream.
```
Comment (Swiftb0y) [inline content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md]:
```suggestion
The official application period will start **April 4**. Potential GSoC contributors should start now to discuss application ideas on our [Zulip](mixxx.zulipchat.com/). Just start a new topic in the [#gsoc stream](https://mixxx.zulipchat.com/#narrow/stream/109215-gsoc).
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Done

File: content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+You will find the [GSoC 2022 Project Ideas for Mixxx](https://github.com/mixxxdj/mixxx/wiki/gsoc-2022-ideas) in our wiki, but we're also interested in project ideas that matters more to you. Taking a look at our [GSoC advice page](https://github.com/mixxxdj/mixxx/wiki/gsocadvice) is recommended if you're planning to apply. The best way to get started is to participate in the Mixxx community, so have a look at our [GitHub repository](https://github.com/mixxxdj/mixxx) and consider making a small code contribution to show that you're able to familiarize yourself with our codebase.
+
+The official application period will start **April 4**. Potential GSoC contributors should start now to discuss application ideas on our [Zulip](mixxx.zulipchat.com/). Just start a new topic in the [#gsoc stream](https://mixxx.zulipchat.com/#narrow/stream/109215-gsoc).
```
Comment (Swiftb0y) [inline content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md]: I'm sorry I proposed a malformed link. Can you amend and force push? 
```suggestion
The official application period will start **April 4**. Potential GSoC contributors should start now to discuss application ideas on our [Zulip](https://mixxx.zulipchat.com/). Just start a new topic in the [#gsoc stream](https://mixxx.zulipchat.com/#narrow/stream/109215-gsoc).
```

Comment (Swiftb0y) [inline content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md]: IMO the application period is quite short. I'd emphasize that. My wording feels a bit off so I'd appreciate if a native speaker (@ywwg) could evaluate whether this sounds ok.

```suggestion
The official application period will start **April 4th** and ends on **April 19th**, so have your applications ready in time. Potential GSoC contributors should start now to discuss application ideas on our [Zulip](https://mixxx.zulipchat.com/). Just start a new topic in the [#gsoc stream](https://mixxx.zulipchat.com/#narrow/stream/109215-gsoc).
```

Comment (daschuer) [reply content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md]: Yes, no problem.

Comment (daschuer) [reply content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md]: I think the comma is missing. I will squash all commits and add the comma.

**Comment by daschuer** [comment]
@Swiftb0y and @ywwg is this OK for merge now? @ywwg: can you also copy it to social media?

**Review by Swiftb0y — APPROVED** [review]
yes, I'd still appreciate if @ywwg would proofread it first before merge.

File: content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+This year the program is open for all new or beginner contributors of Mixxx. It is no longer limited to enrolled students. New contributors have the opportunity to work on Mixxx full-time as a 12+ weeks programming project and receive a stipend sponsored by Google. For more information, check out the [Google Summer of Code Contributor Guide](https://google.github.io/gsocguides/student/).
+
+You will find the [GSoC 2022 Project Ideas for Mixxx](https://github.com/mixxxdj/mixxx/wiki/gsoc-2022-ideas) in our wiki, but we're also interested in project ideas that matters more to you. Taking a look at our [GSoC advice page](https://github.com/mixxxdj/mixxx/wiki/gsocadvice) is recommended if you're planning to apply. The best way to get started is to participate in the Mixxx community, so have a look at our [GitHub repository](https://github.com/mixxxdj/mixxx) and consider making a small code contribution to show that you're able to familiarize yourself with our codebase.
```
Comment (ywwg) [inline content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md]:
```suggestion
You will find the [GSoC 2022 Project Ideas for Mixxx](https://github.com/mixxxdj/mixxx/wiki/gsoc-2022-ideas) in our wiki, but we're also interested in project ideas that matter more to you. Taking a look at our [GSoC advice page](https://github.com/mixxxdj/mixxx/wiki/gsocadvice) is recommended if you're planning to apply. The best way to get started is to participate in the Mixxx community, so have a look at our [GitHub repository](https://github.com/mixxxdj/mixxx) and consider making a small code contribution to show that you're able to familiarize yourself with our codebase.
```

**Review by ywwg — APPROVED** [review]
just one tiny pluralization mismatch, otherwise it sounds good!

**Comment by daschuer** [comment]
Done, Thank you.

**Review by Swiftb0y — APPROVED** [review]

**Pull request merged**